tests/protractor/utils.js

```diff
@@ -54,57 +54,57 @@ var request = function(options, debug) {
-      throw new Error('A previous test did not call revertSettings on ' + ddocName);
-    }
-    return request({
-      path: path.join('/', constants.DB_NAME, '_design', mainDdocName, '_rewrite/app_settings', ddocName),
-      method: 'GET'
-    }).then(function(result) {
-      originalSettings[ddocName] = result.settings;
-
-      // Make sure all updated fields are present in originalSettings[ddocName], to enable reverting later.
-      Object.keys(updates).forEach(function(updatedField) {
-        if (!_.has(originalSettings[ddocName], updatedField)) {
-          originalSettings[ddocName][updatedField] = null;
-        }
-      });
-      return;
-    }).then(function() {
-      return request({
-        path: path.join('/', constants.DB_NAME, '_design', mainDdocName, '_rewrite/update_settings', ddocName, '?replace=1'),
-        method: 'PUT',
-        body: JSON.stringify(updates)
-      });
+var updateSettingsForDdoc = function (updates, ddocName) {
+  if (originalSettings[ddocName]) {
+    throw new Error('A previous test did not call revertSettings on ' + ddocName);
+  }
+  return request({
+    path: path.join('/', constants.DB_NAME, '_design', mainDdocName, '_rewrite/app_settings', ddocName),
+    method: 'GET'
+  }).then(function (result) {
+    originalSettings[ddocName] = result.settings;
+
+    // Make sure all updated fields are present in originalSettings[ddocName], to enable reverting later.
+    Object.keys(updates).forEach(function (updatedField) {
+      if (!_.has(originalSettings[ddocName], updatedField)) {
+        originalSettings[ddocName][updatedField] = null;
+      }
     });
-  };
-
-var revertSettingsForDdoc = function(ddocName) {
-    if (!originalSettings[ddocName]) {
-      throw new Error('No original settings to revert to for ' + ddocName);
-    }
-
+    return;
+  }).then(function () {
     return request({
       path: path.join('/', constants.DB_NAME, '_design', mainDdocName, '_rewrite/update_settings', ddocName, '?replace=1'),
       method: 'PUT',
-      body: JSON.stringify(originalSettings[ddocName])
-    }).then(function() {
-      delete originalSettings[ddocName];
+      body: JSON.stringify(updates)
     });
-  };
+  });
+};
+
+var revertSettingsForDdoc = function (ddocName) {
+  if (!originalSettings[ddocName]) {
+    throw new Error('No original settings to revert to for ' + ddocName);
+  }
+
+  return request({
+    path: path.join('/', constants.DB_NAME, '_design', mainDdocName, '_rewrite/update_settings', ddocName, '?replace=1'),
+    method: 'PUT',
+    body: JSON.stringify(originalSettings[ddocName])
+  }).then(function () {
+    delete originalSettings[ddocName];
+  });
+};
 
 
 module.exports = {
 
   request: request,
 
-  requestOnTestDb: function(options, debug) {
+  requestOnTestDb: function (options, debug) {
     options.path = '/' + constants.DB_NAME + options.path;
     return request(options, debug);
   },
 
-  saveDoc: function(doc) {
+  saveDoc: function (doc) {
```

Comment:
It looks like your formatter is configured to add spaces after the `function` keyword which isn't consistent with how our code is usually formatted. Can you try to configure your formatter to be consistent with our existing code?